### ISSUE TITLE
Instructions to getting 'whois' command in readme

### ISSUE THREAD

**Issue by Elyrith** [glensc/nagios-plugin-check_domain#22]
TL:DR - Is it worth putting whois package install instructions in when it's quick and easy to Google?

Early in 2014 I rented a fresh CentOS 6 machine and it did not come with the 'whois' installed. It was easy to look up how to do this, but is it worthwhile to put this type of info into README.md?

Normally I just put something like this in my plugins, but it's just an example:

This plugin uses the "whois" command. It is usually installed by default, but if not, you can get it via your favourite package manager:
Debian/Ubuntu: apt-get install whois
RHEL/CentOS: yum install jwhois

If this makes sense, I can create a PR, but I wanted to ask first.

**Comment by glensc** [comment]
yeah. i do not mind. thanks!
